package/kernel/r8126/patches/0002-print-link-speed-and-duplex-mode.patch

```diff
@@ -0,0 +1,37 @@
+From cf8d0528db1959ae13a1ba3240e06877f06d2b16 Mon Sep 17 00:00:00 2001
+From: Chukun Pan <amadeus@jmu.edu.cn>
+Date: Sat, 10 Aug 2024 20:16:32 +0800
+Subject: [PATCH] r8126: print link speed and duplex mode
+
+Like other Ethernet drivers, print link speed and duplex mode
+when the interface is up. Formatting output at the same time.
+
+Signed-off-by: Chukun Pan <amadeus@jmu.edu.cn>
+---
+ src/r8126_n.c | 8 ++++++--
+ 1 file changed, 6 insertions(+), 2 deletions(-)
+
+--- a/src/r8126_n.c
++++ b/src/r8126_n.c
+@@ -4744,15 +4744,19 @@ static void
+ _rtl8126_check_link_status(struct net_device *dev)
+ {
+         struct rtl8126_private *tp = netdev_priv(dev);
++        u16 status = RTL_R16(tp, PHYstatus);
```

Comment:
AFAIK this could be moved under if (netif_msg_ifup(tp)) because we're only using it there.